pkgs/applications/misc/holochain-launcher-bin/default.nix

```diff
@@ -0,0 +1,57 @@
+{ stdenv
+, lib
+, fetchurl
+, autoPatchelfHook
+, pkg-config
+, dpkg
+, openssl
+, webkitgtk
+, libappindicator
+, wrapGAppsHook
+}:
+
+let
+  version = "0.5.1";
+in
+
+stdenv.mkDerivation {
+  name = "holochain-launcher-bin";
+  inherit version;
+
+  src = fetchurl {
+    url = "https://github.com/holochain/launcher/releases/download/v${version}/holochain-launcher_${version}_amd64.deb";
+    sha256 = "sha256-SJzDQQ3ltieZEoLsqEGnXskQcc+0n0Dwu36w59G87Qg=";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    pkg-config
```

Comment:
Why do we need pkg-config when not building from source?

Reply:
thanks for catching that, removed